ivy_tests/test_ivy/test_frontends/test_jax/test_random.py

```diff
@@ -1355,6 +1355,68 @@ def call():
         assert u.shape == v.shape
 
 
+@pytest.mark.xfail
+@handle_frontend_test(
+    fn_tree="jax.random.double_sided_maxwell",
+    dtype_key=helpers.dtype_and_values(
+        available_dtypes=["uint32"],
```

Comment:
You are specifying the dtype here, this should be `valid` dtype and the dtypes that should be skipped should be added in the decorator above.

Reply:
and pass in `valid` here, so that the decorator skips the dtypes which are not to be tested on. Rather than us trying to restrict those in the tests.